### PR TITLE
book library

### PR DESCRIPTION
**Pull request by dadu99**
Book library which includes authors and books. On these cagetories CRUD operations can be done
All functionalities are made in Node.js, Express.js and EJS JavaScript Templates

### CODE REVIEW

File: views/partials/errorMessage.ejs
Comment (BadeaCristina) [inline views/partials/errorMessage.ejs:1]: may be able to use bootstrap alert..

File: README.md
Comment (BadeaCristina) [inline README.md:1]: more explanations to readme and please add image with whole app

File: views/layouts/layout.ejs
[the remark this answers is not in the document]
Comment (dadu99) [reply views/layouts/layout.ejs:1]: well said

[the remark this answers is not in the document]
Comment (dadu99) [reply views/layouts/layout.ejs:1]: done :)

**Pull request merged**